### ISSUE TITLE
Symfony Console and CLImate not outputting correct colors to the terminal

### ISSUE THREAD

**Issue by joemottershaw** [zeit/hyper#2597]
I'm using [Symfony Console](http://symfony.com/doc/current/components/console.html) and using their [color styles](https://symfony.com/doc/current/console/coloring.html#using-color-styles) to output colored text in a PHP command script I'm writing.

However, I'm just getting the default white text outputted rather than the corresponding color described in their documentation. I would just create an issue there but I've also tried [CLImate](https://github.com) by [thephpleague](https://github.com/thephpleague/) and used the code described in their documentation and I'm still getting the same issue, so if 2 different console packages are failing I'm guessing it's something I'm doing wrong...

I've also tried various themes within Hyper and can't get it to work so I'm guessing I've most likely got something setup wrong with my configuration within Hyper?

I've linked to my hyper.js file in a Gist below to show my settings/plugins, would just like to know if it is something wrong on my end, Symfony Console or CLImate (and I can raise an issue there) or if I'm missing something I need to do within Hyper itself?

- **OS**: Windows 10 Home, Version 1702, Build 15063.850
- **Hyper**: 1.4.8
- **Gist**: [hyper.js](https://gist.github.com/joemottershaw/0e3f1e90081dd5ba526a86cc6f70022d)

**Comment by joemottershaw** [comment]
Yes, fresh install of Hyper (2.0.0 - no plugins/themes) on the same PC (updated Windows from the last post).

I created a 'climate' directory in the MAMP htdocs directory for demo purposes just now. Used composer to require the latest CLImate package into the vendor directory of the 'climate' directory using `composer require league/climate` in the Hyper terminal and then created an `index.php` file with the following script:

```
<?php

require_once __DIR__ . "/vendor/autoload.php";

$climate = new \League\CLImate\CLImate;

$climate->red('Whoa now this text is red.');
$climate->blue('Blue? Wow!');

```

Back into Hyper I ran the script `php index.php`, the text was printed out but in all white.

![hyper](https://user-images.githubusercontent.com/5093255/39871820-f74e31e2-545d-11e8-8980-c35a30f1d422.png)
